src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.model;
+
+import core.basesyntax.model.enums.Operation;
+
+public class FruitTransaction {
```

Comment:
Don't begin class or method implementation with empty line. Remove the redundant empty line at the start of the class definition.